src/sinks/datadog/logs/sink.rs

```diff
@@ -23,9 +23,10 @@ pub struct EventPartitioner;
 impl Partitioner for EventPartitioner {
     type Item = Event;
     type Key = Option<Arc<str>>;
```

Comment:
TODO: check if we still want to use Option here or if this is missing this is a failure